src/server/gdb_server.c

```diff
@@ -3016,7 +3016,7 @@ static bool gdb_handle_vcont_packet(struct connection *connection, const char *p
 	__attribute__((unused)) int packet_size)
```

Comment:
It seems to me that the GDB's `c` packet would require the same treatment (`gdb_step_continue_packet()`).

Or, is GDB not using the plain `c` packet (in favor of `vCont:c`)? If that is the case, would you know if this is GDB's general behavior or only on multi-threaded targets?

Reply:
I've never seen gdb use a plain 'c' packet. I assume that if vCont is supported, then it always uses that.

Reply:
> I've never seen gdb use a plain 'c' packet. I assume that if vCont is supported, then it always uses that.

Do you ever use, say, an Eclipse or VSCode based environment for debugging? I just wonder if that sort of setup, that usually uses the GDB/MI rather than the regular CLI to drive GDB, might cause `c` packets to be used in some cases? Wouldn't it be safer to apply the same changes to the OpenOCD `c` packet handler just in case?

Reply:
The current merge request dealing with `vCont` is OK with me. This was a note to keep the back of one's mind, just in case the `c` packet should ever occur and started causing troubles.

Reply:
Reading the code for handling 'c' packets, I'm not convinced it does anything at all. I'm not going to attempt to address that unless I see a problem report.

@TommyMurphyTM1234 I don't use Eclipse, but I do sometimes go over -d3 logs from people who do.